Bosworth Toller's Anglo-Saxon Dictionary Online.js

```diff
@@ -0,0 +1,636 @@
+				"url": "https://bosworthtoller.com/7096",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [],
+				"notes": [
+					{
+						"note": "<h1>Summary</h1>CYN<h2>Word category</h2><ul><li>noun [ masculine, neuter ]</li></ul><h2>Grammar</h2>CYN, cynn,es; n.<h2>Definition</h2><ul><li>every being of one kind, a kindred, kind, race, nation, people, tribe, family, lineage, generation, progeny, KIN ; genus, gens, natio, populus, stirps, tribus, familia, natales, origo, generatio, proles, progenies</li><li>Gender; genus</li><li>a sex; sexus</li></ul>"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://bosworthtoller.com/27305",
+		"detectedItemType": "dictionaryEntry",
+		"items": [
+			{
+				"itemType": "dictionaryEntry",
+				"title": "secgan",
+				"creators": [
+					{
+						"firstName": "Joseph",
+						"lastName": "Bosworth",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Thomas Northcote",
+						"lastName": "Toller",
+						"creatorType": "editor"
+					},
+					{
+						"firstName": "Christ",
+						"lastName": "Sean",
+						"creatorType": "editor"
+					},
+					{
+						"firstName": "Ondřej",
+						"lastName": "Tichy",
+						"creatorType": "editor"
+					}
+				],
+				"date": "2014",
+				"dictionaryTitle": "An Anglo-Saxon Dictionary Online",
+				"language": "en",
+				"libraryCatalog": "Bosworth Toller's Anglo-Saxon Dictionary Online",
+				"place": "Prague",
+				"publisher": "Faculty of Arts, Charles University",
+				"rights": "All the data provided here are free for any purpose. If you use the data for academic purposes, we ask that you kindly cite us as your source. In case you need a large portion of the dictionary data or you need it in a specific format, let us know through the contact form, we are happy to do custom database dumps for researchers.",
+				"url": "https://bosworthtoller.com/27305",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [],
+				"notes": [
+					{
+						"note": "<h1>Summary</h1>secgan<h2>Word category</h2><ul><li>verb [ weak ]</li></ul><h2>Grammar</h2>secgan, secgean, secggan, secggean, sæcgan ; p. sægde, sǽde; pp. sægd, sǽd. [Forms as from an infin. sagian—sagast, sagaþ ; p. sagode; imp. saga, are given here.]<h2>Definition</h2><ul><li>To say (of written or spoken words).</li><li>to say certain words, the words used being given</li><li>pronounce, deliver</li><li>to speak of, tell, relate, narrate, declare, announce, give an account of something</li><li>Ger. Dank sagen)</li><li>to declare a person or thing so and so</li><li>to say, tell</li><li>Icel. segir it is told)</li><li>to speak of)</li><li>lay to the charge of, accuse of, attribute to</li></ul>"
```

Comment:
The notes looks good to me (see my note about ol/ul above). Do they get much longer than this? At some point that becomes undesirable, but I think in this length they're great.